src/domain/community/user/userContributions/useUserContributions.ts

```diff
@@ -0,0 +1,46 @@
+import { useUserContributionsQuery } from '../../../../core/apollo/generated/apollo-hooks';
+import { useUserContext } from '../hooks/useUserContext';
+import { ContributionItem } from '../contribution';
+import { useMemo } from 'react';
+
+const useUserContributions = () => {
+  const { user } = useUserContext();
+
+  const { data } = useUserContributionsQuery({
+    variables: {
+      userId: user?.user.id!,
+    },
+    skip: !user?.user.id,
+  });
+
+  return useMemo(() => {
+    if (!data) {
+      return undefined;
+    }
+
+    const spaces = data?.rolesUser.spaces.map<ContributionItem>(e => ({
+      spaceId: e.nameID,
```

Comment:
Should this be ID or nameID? Either the value or the name I think needs updating?